### PR TITLE
[simple-app] Re-order HPA metrics

### PR DESCRIPTION
**Pull request by ericayin**
https://github.com/kubernetes/kubernetes/issues/74099

### CODE REVIEW

File: charts/simple-app/values.yaml [hunk 1]
```diff
@@ -342,7 +342,7 @@ autoscaling:
   # -- Configures the HPA to target a particular CPU utilization percentage
   targetCPUUtilizationPercentage: 80
   # -- If supplied, configures the HPA to target a particular Memory utilization percentage
-  # targetMemoryUtilizationPercentage: 80
```
Comment (diranged) [inline charts/simple-app/values.yaml:345]: Eh. I think it should be uncommented, but also left blank. You don't generally want to autoscale on CPU and Memory in _most_ cases without lots of tuning. That's why it was left off here. It doesn't change the capability of the downstream chart from setting the value though.

Comment (ericayin) [reply charts/simple-app/values.yaml]: ooh yea that makes sense, i'll leave it as is

**Review by diranged — APPROVED** [review]
:+1: thank you!

File: charts/simple-app/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: simple-app
 description: Default Microservice Helm Chart
 type: application
-version: 0.16.2
+version: 0.16.3
```
Comment (diranged) [inline charts/simple-app/Chart.yaml:5]: @ericayin oh rebase.. we're on 0.17+ now..

**Pull request merged**